content/maxwell3_fdem/emi_tutorial/amplitudeandphase.rst

```diff
@@ -0,0 +1,129 @@
+.. _amplitudeandphase:
+
+Amplitude and phase
+===================
+
+.. figure:: ./images/Concepts_3loops_only_K.png
+   :align: center
+   :scale: 60%
+   :name: Concepts_3loops_only_K
+
+   Conceptual diagram for 3-loops system.
+
+Based upon the induced current in the body (Loop2), in :ref:`induced_currents_body`, here we consider amplitude and phase of the secondary magnetic field (:math:`H^s`) normalized by primary magnetic field (:math:`H^p`), :math:`\frac{H^s}{H^p}`, measured at Rx (Loop3). For this, we first consider the primary magnetic field at body (Loop2), :math:`H^p_2`, resulting from a current, :math:`i^p`, flowing in the Loop1, which can be expressed as
+
+.. math::
+	H^p_2(t) = K_{12} i^p(t),
+
+where :math:`i^p = I^p cos(\omega t)` and :math:`K_{12}` is coefficint depending on the geomtry of the system, the area, and number of turns of the primary coil, and attenuation of the wave between Loop1 and Loop2.
+
+Effectively, through Faraday's law, :math:`\mathcal{E} = -\imath \omega \Phi^p`, volatage will be generated, which generates induced currents (this has been done in :ref:`induced_currents_body`):
+
+.. math::
+    I(\omega) = \frac{-\imath \alpha}{1+\imath\alpha}\frac{\Phi^p}{L},
+
+where :math:`\alpha=\frac{\omega L}{R}`. EMF induced here can be written as
+
+.. math::
+	\mathcal{E} = -M\frac{d i^p}{dt} = -M \imath \omega I^p = -M \imath \omega \frac{H^p_2}{K_{12}},
+	:label: EMF
+
+where :math:`M` is mutual inductance, and this property illustrates how much voltage induced in Loop2 due to time varying current in Loop1. This yields :math:`\Phi^p = \frac{\mathcal{E}}{-\imath \omega} = M\frac{H^p_2}{K_{12}}`. The secondary magnetic field, :math:`H_s` at the Rx (Loop3) due to this current will be
+
+.. math::
+	H^s =K_{23} I(\omega)
```

Comment:
Could you tie the equations to the figure more (ie, highlight where H^p, H^s are, similar to the following, but also showing the geometric terms)? Connecting the math with things in a diagram is really helpful 
<img width="665" alt="induced currents in the body em 0 0 1 documentation 2016-04-12 21-10-35" src="https://cloud.githubusercontent.com/assets/6361812/14482627/6476ef2c-00f3-11e6-89a1-198a5a822a78.png">